generated/niscope/niscope_library.cpp

```diff
@@ -580,7 +580,7 @@ ViStatus NiScopeLibrary::Disable(ViSession vi)
 #endif
 }
 
-ViStatus NiScopeLibrary::ErrorHandler(ViSession vi, ViStatus errorCode, ViChar errorSource[642], ViChar errorDescription[642])
+ViStatus NiScopeLibrary::ErrorHandler(ViSession vi, ViStatus errorCode, ViChar errorSource[55], ViChar errorDescription[642])
```

Comment:
Neither of these buffer sizes make sense to me. Character arrays of a fixed size are usually 256 or something like that. Thoughts?